Cargo.toml

```diff
@@ -20,6 +20,7 @@ etcd-rs = "0.2.2"
 clap = "2.33.3"
 flate2 = "1.0.16"
 futures = "0.3.5"
+get_if_addrs = "0.5.3"
```

Comment:
do not use archived crate

Reply:
How about https://github.com/libpnet/libpnet?